package.json

```diff
@@ -14,7 +14,7 @@
     "moment": "^2.24.0",
     "node-html-parser": "^1.2.12",
     "query-string": "^6.12.0",
-    "rasa-webchat": "^0.8.8",
+    "rasa-webchat": "git://github.com/nuecho/rasa-webchat#v0.9.0+nuecho.1",
```

Comment:
I know, once again not ideal, but will have to do for now. It points to a gti tag own our own fork of the widget.
We submitted a fix to the project but are still waiting on them to merge it.